modules/installation-creating-azure-service-principal.adoc

```diff
@@ -45,9 +41,9 @@ control. For more information, see https://aka.ms/azadsp-cli
 }
 ----
 
-. Record the values of the `appId` and `password` parameters from the output. You require these values when installing the cluster.
+Record the values of the `appId` and `password` parameters from the output. You require these values when installing the cluster.
 
-. If you applied the `Contributor` role to your service principal, assign the `User Administrator Access` role by running the following command:
```

Comment:
The sequence looks changed here, starting from 1 here when reviewing doc, https://79423--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-account.html#installation-creating-azure-service-principal_installing-azure-account